src/libs/auth.js

```diff
@@ -192,7 +197,7 @@ authStore.subscribe(withErrorReporting('Error checking TOS', async (state, oldSt
   }
 }))
 
-authStore.subscribe(async (state, oldState) => {
+authStore.subscribe(withErrorIgnoring(async (state, oldState) => {
```

Comment:
Taking this opportunity to wrap more Ajax calls with error handling to prevent potential sporadic and difficult to diagnose test failures.

Reply:
Asking to understand: Why is ignoring the desirable behaviour?

Reply:
I don't think we want to show the error dialog to the user if these ajax calls fail-- they are things we are doing "behind the scenes" that aren't critical to folks using Terra. At least, that's what I'm reading into the fact that there was no error handling around them.

Reply:
@breilly2 do you have any insight here? It looks like you, Isaac, and Brett wrote most of this code.